hooks/useLobbyActions.ts

```diff
@@ -11,6 +9,8 @@ const database = getDatabase();
 export default function useLobbyActions() {
   const router = useRouter();
   const { roomId } = router.query;
+  let canBeStarted = true;
```

Comment:
These are just variables, should have been inside the onStartGame function not here.